packages/block-editor/src/components/inserter/tabs.js

```diff
@@ -29,23 +33,33 @@ const mediaTab = {
 	title: __( 'Media' ),
 };
 
-function InserterTabs( { onSelect, children }, ref ) {
+function InserterTabs( { onSelect, children, onClose }, ref ) {
 	const tabs = [ blocksTab, patternsTab, mediaTab ];
 
 	return (
 		<div className="block-editor-inserter__tabs" ref={ ref }>
 			<Tabs onSelect={ onSelect }>
-				<Tabs.TabList className="block-editor-inserter__tablist">
-					{ tabs.map( ( tab ) => (
-						<Tabs.Tab
-							key={ tab.name }
-							tabId={ tab.name }
-							className="block-editor-inserter__tab"
-						>
-							{ tab.title }
-						</Tabs.Tab>
-					) ) }
-				</Tabs.TabList>
+				<div className="editor-inserter-sidebar__header">
```

Comment:
```suggestion
				<div className="block-editor-inserter-sidebar__header">
```

Reply:
Done in [7092ac1](https://github.com/WordPress/gutenberg/pull/61421/commits/7092ac1456c0238a9f9e1576c5839260699ef3b0)